kb/2023-12-27/best_practices_disk_performance.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Best Practices for Optimizing Longhorn Disk Performance
+description: Follow the recommendations for achieving optimal disk performance.
+slug: best_practices_for_optimizing_longhorn_disk_performance
+authors:
+  - name: David Ko
+    title: Senior Software Engineering Manager
+    url: https://github.com/innobead
+    image_url: https://github.com/innobead.png
+  - name: Jillian Maroket
+    title: Technical Writer
+    url: https://github.com/jillian-maroket/
+    image_url: https://github.com/jillian-maroket.png
+tags: [harvester, longhorn, best practices, disk performance]
+hide_table_of_contents: false
+---
+
+# Best Practices for Optimizing Longhorn Disk Performance
+
+The Longhorn documentation provides [best practice recommendations](https://longhorn.io/docs/1.5.3/best-practices/) for deploying Longhorn in production environments. Before configuring workloads, ensure that you have set up the following basic requirements for optimal disk performance.
+
+- SATA/NVMe SSDs or disk drives with similar performance
+- 10 Gbps network bandwidth between nodes
+- Dedicated Priority Classes for system-managed and user-deployed Longhorn components
+
+The following sections outline other recommendations for achieving optimal disk performance.
+
+## IO Performance
+
+- **Storage network**: Use a dedicated storage network to improve IO performance and stability.  
+
+- **Longhorn disk**: Use a dedicate for Longhorn storage instead of using the root disk.  
+
+- **Replica count**: Set the [default replica count](https://longhorn.io/docs/1.5.3/references/settings/#default-replica-count) to "2" to achieve data availability with better disk space usage or less impact to system performance. This practice is especially beneficial to data-intensive applications.  
```

Comment:
We need to provide the harvester way here. To create a storage class with the replica count, and create VM images and volumes from it: https://docs.harvesterhci.io/v1.2/advanced/storageclass#parameters-tab